tabbycat/standings/ranking.py

```diff
@@ -18,6 +18,18 @@
 logger = logging.getLogger(__name__)
 
 
+def _generate_ordering(metrics, annotators, min_field, min_rounds):
+    ordering = []
+    annotations = {a.key: a for a in annotators}
+    for key in metrics:
+        annotation = annotations[key]
+        if annotation.ascending:
+            ordering.append(annotation.get_ranking_annotation(min_field, min_rounds).asc(nulls_last=True))
+        else:
+            ordering.append(annotation.get_ranking_annotation(min_field, min_rounds).desc(nulls_last=True))
+    return ordering
```

Comment:
How about have this as a `@staticmethod` inside `BaseRankAnnotator`, keeping the `_get_ordering()` name? Then we wouldn't need the methods to merely call this function. As a result, line 167 would look something like (would also need to modify the other places calling `_get_ordering()`):
```py
order_by=self._get_ordering(self.metrics[1:], annotators, min_field, min_rounds),
```